src/components/Projects.tsx

```diff
@@ -1,7 +1,247 @@
 import React from "react";
+import styled from "styled-components";
+import { Link } from "react-router-dom";
+import {
+  space,
+  SpaceProps,
+  layout,
+  LayoutProps,
+  background,
+  BackgroundProps,
+  FlexboxProps,
+  flexbox,
+  PositionProps,
+  position,
+  ColorProps,
+  color,
+  TypographyProps,
+  typography,
+  border,
+  BorderProps,
+} from "styled-system";
+import stairs from "./assets/project-page/stairs.png";
+import shell from "./assets/project-page/shell.png";
+import eye from "./assets/project-page/eye.png";
+import statue from "./assets/project-page/statue.png";
+import dragon from "./assets/project-page/dragon.png";
+import knife from "./assets/project-page/knife.png";
+import mask from "./assets/project-page/mask.png";
+import spider from "./assets/project-page/spider.png";
+import magnify from "./assets/project-page/magnify.png";
+
+type BgProps = BackgroundProps &
+  ColorProps &
+  LayoutProps &
+  FlexboxProps &
+  TypographyProps &
```

Comment:
Typography props arent used by the Background component

Reply:
Very true